### DISCUSSION TITLE
Issue with Expense accounts' names in the graph

### DISCUSSION THREAD

**Discussion by steakhutzeee** [firefly-iii/firefly-iii#4381] [General]
I am running Firefly III version 5.4.6.

**Description**

When on mobile view not all the Expense accounts'names are visibile below the graph. Switching to desktop view fix this. No issue with the Categories graph.

Attached a screenshot. See that one is shown and one is not, and so on.

![screen](https://user-images.githubusercontent.com/57228102/107561572-9a402300-6bd6-11eb-8e5f-f37a9eb03bb9.jpg)

**Comment by JC5** [comment]
Thanks for letting me know. I'm not going to fix it unfortunately. I'm working on a new layout where this issue will be gone.
